README.md

```diff
@@ -0,0 +1,41 @@
+# Encryptonize&reg; library
+[![Library build](https://github.com/cyber-crypt-com/encryptonize-lib/actions/workflows/build.yaml/badge.svg)](https://github.com/cyber-crypt-com/encryptonize-lib/actions/workflows/build.yaml)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cyber-crypt-com/encryptonize-lib)](https://goreportcard.com/report/github.com/cyber-crypt-com/encryptonize-lib)
+[![Go Reference](https://pkg.go.dev/badge/github.com/cyber-crypt-com/encryptonize-lib)](https://pkg.go.dev/github.com/cyber-crypt-com/encryptonize-lib)
+
+The Encryptonize&reg; library provides cryptographic functions for data encryption and decryption, as well as enforcing access control around the encrypted data. It allows developers to easily implement various security schemes and protocols for protecting data.
+
+## Features
+- **Authenticated Encryption with Associated Data (AEAD)**
+    - binary data can be encrypted/decrypted using the standardized AES256-GCM algorithm
+    - the Ciphertext and Associated Data can be securely stored in untrusted locations as the recovery of the plaintext is computationally infeasible without the Encryption Key, and any changes to the Ciphertext or Associated Data will be detected during the decryption process
+- **Key Wrapping with Padding Authenticated Encryption (KWP-AE)**
```

Comment:
KWP-AE is a algorithm, whereas AEAD in the headline above is a concept.

Reply:
Fixed. I left only the concepts in the subtitles.